protostar/commands/deploy/deploy_command.py

```diff
@@ -61,6 +60,12 @@ def arguments(self) -> List[Command.Argument]:
                 type="str",
                 is_array=True,
             ),
+            Command.Argument(
+                name="build-output",
```

Comment:
It probably should be source-dir, I first thought we are building contract inside this command and was weirded out

Reply:
Definity it shouldn't be named source-dir. It suggest pointing to src directory.

Reply:
Maybe the user should simply pass a path(s) to the compiled contract.

Reply:
True, maybe `compiled-contract-dir`?